commitlint.config.ts

```diff
@@ -0,0 +1,30 @@
+import type { UserConfig } from "@commitlint/types";
+
+const scopeEnumValues = [
+  "base",
+  "bi",
+  "ckb-indexer",
+  "common-scripts",
+  "config-manager",
+  "hd",
+  "hd-cache",
+  "helper",
+  "indexer",
+  "lumos",
+  "rpc",
+  "sql-indexer",
+  "testkit",
+  "toolkit",
+  "transaction-manager",
+  "examples",
+  "website",
+];
+const Configuration: UserConfig = {
+  extends: ["@commitlint/config-conventional"],
+  rules: {
+    "scope-enum": [2, "always", scopeEnumValues],
+    "scope-empty": [2, "never"],
```

Comment:
maybe we should NOT ban an empty scope, just like this commit, `chore: lint for commit message` is just fine

Reply:
In order to prevent some one forget to add scope which should be added, I set scope-empty not allowed.
and for global commit message we can add `lumos` instand.

Reply:
removed scope-empty rule